default.nix

```diff
@@ -570,42 +671,47 @@
     debug-env = toolchainPkgs.dockerTools.buildLayeredImage {
       name = "${contianer-repo}/debug-env";
       tag = "${image-tag}";
-      contents = (with toolchainPkgs; [
-        bashInteractive
-        coreutils
-        curl
-        debianutils
-        ethtool
-        gawk
-        gdb
-        gnugrep
-        gnused
-        iproute2
-        iptables
-        jq
-        less
-        nano
-        procps
-        rr
-        strace
-        tcpdump
-        tshark
-        valgrind
-        vim
-        zstd
-      ]) ++ (with pkgs.${profile}.gnu64; [
-        glibc.out
-        glibc.bin
-        libgcc.libgcc
-        perftest
-      ]);
+      contents =
+        (with toolchainPkgs; [
```

Comment:
Content for this block changed, too

Reply:
looks like valgrind was removed and dmidecode was added.  Will fix